README.md

```diff
@@ -1,2 +1,27 @@
-# xdd
-Aliases as a service
+# Hivecom Emotes (codename: xdd)
+
+Is an emote management and provider system for other hivecom projects. It allows registered users to create, edit and share emotes which can then be used in other applications which involve user input. Such as [hifriends](https://friends.hivecom.net/) or [quotes](https://quotes.hivecom.net/).
+
+The app uses authentication and permission system provided by our own authentication system [idlib](https://github.com/Mavulp/hiveID).
```

Comment:
I'm not sure how interesting this is to someone reading the readme. It would probably make more sense to have it further down and keep the description and images of what this project does at the top. I think it is relevant but I wouldn't care about this information until after I'm interested in the project itself.